### ISSUE TITLE
Add all audio bitrates

### ISSUE THREAD

**Issue by phly95** [TeamNewPipe/NewPipe#1059]
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I checked if the issue/feature exists in the latest version.

So I recently ran out of LTE data, and it seems I have about 128k speeds. So while it's barely enough to keep up with just audio, add video in the mix and it becomes too heavy. I noticed however, that 3GP video contains lower bitrate audio, and when including both 3GP video and audio, it is half as much as just the 128k stream. The MP4 144p option when downloaded appears to be smaller than the 3GP file but it doesn't include audio, making newpipe retrieve the higher quality 128k stream, thereby tripling the required data to stream.

Basically, I would like the ability to use the crappier low bitrate audio and be able to combine it with MP4 144p video, as well as being able to use the crappy audio as standalone for those really slow connections.

**Comment by theScrabi** [comment]
Hm tha sounds like some useful feature.

**Comment by Cormak** [comment]
@theScrabi To be perfectly honest, you should just add ALL the audio bitrates and formats that are possible to add. 

This will make Newpipe way more flexible and powerful. A lot of people like to download the audio from LONG videos and then listen to it using an audio player so we don't have to stream it again and not lose our place. Adding all the bitrates that are possible will allow us to get low quality audio for basic spoken word stuff and high quality audio for music stuff.

You could merge this issue in as well https://github.com/TeamNewPipe/NewPipe/issues/837

And then fix all these issues with a PR that exposes ALL possible audio bitrates/formats.

**Comment by wb9688** [comment]
Related: TeamNewPipe/NewPipeExtractor#71
